web/packages/teleterm/src/services/config/getKeyboardShortcutSchema.ts

```diff
@@ -68,6 +69,16 @@ function putModifiersFirst(
     });
 }
 
+/** Currently works only for single characters. */
```

Comment:
I think I commented on this but maybe I didn't submit my comment?

I wanted to say that in the comment we should also explain why it works only for single characters.

Reply:
I think I saw it, added a longer explanation.